docs/admin.rst

```diff
@@ -197,25 +196,6 @@ possession when they attempt to log in to SecureDrop.
 .. |Verify YubiKey|
   image:: images/manual/screenshots/journalist-admin_new_user_two_factor_hotp.png
 
-.. _Updating the Workstations:
-
-Updating SecureDrop Workstations
---------------------------------
-
-In addition to Tails updates, once you have upgraded to SecureDrop 0.6 or later,
-you can keep the SecureDrop software up to date on each *Journalist Workstation*
-and *Admin Workstation* using the ``securedrop-admin`` CLI. In each workstation:
-
-.. code:: sh
-
-  cd ~/Persistent/securedrop
-  ./securedrop-admin setup  # Run setup before the update
-  ./securedrop-admin update
-  ./securedrop-admin setup  # Run setup again after the update
```

Comment:
Thought: We should be running `securedrop-admin setup` in the future as part of the automatic update just in case a dependency is updated

Reply:
True! In 0.7, we introduced a new Ansible dependency, `netaddr` as part of #2592.

Reply:
:+1: